### DISCUSSION TITLE
[Feature request] hashtags in notes automatically adding tags

### DISCUSSION THREAD

**Discussion by pretentious7** [standardnotes/forum#2612] [Suggestions] [answered]
Hello,

A feature I use a lot in Google keep is auto adding tags using hashtags within notes. It would be cool to see this implemented.

Thank you.

**Comment by bryvin** [comment]
Due to needing to support Markdown title syntax (with the Markdown editors) and likely a lot of other formats people use within their notes, adding auto tagging based on the note's content may produce unexpected results for most users. This is also a technical challenge as each of our editors has a different source and this would have to be implemented into each editor.

I'll move this thread to the forum so it can remain for discussion.

**Comment by pretentious7** [comment]
Here's the thing I'd just script it if it were the desktop app but on mobile I'd have to do a bunch of things and make an apk and maintain it and it all just seems like a pita. Are you aware of any hacky solutions that would just read when I'm inputting text into a file and autorun the add tag?

**Comment by pretentious7** [comment]
I mean notwithstanding I'm going to switch to standard notes or joplin to get away from google, but it'd be really nice to have this figured out before I do cause I really do use it a lot.

**Comment by arielsvg** [comment]
I think this is as much a design challenge as it is a technical one 🤔 There may be ways to cover your use case and a bunch of other ones with one functionality/extension! We'll forward your request & use-case to everyone, thanks for the suggestion 😃

**Comment by myreli** [comment, marked as answer]
While it doesn't use hashtags, the [new Super editor](https://www.youtube.com/watch?v=bkp6nrGqjUM&t=370s) supports this through typing `@`, which you can use to embed tags, files or even other notes.
